Followers RnD addition

SquishyOranges opened the pull request:
## About The Pull Request

Adds Research to the Follower's Underground area.

## Why It's Good For The Game
It simply allows the Followers to treat their patients more efficiently and allows them to do what they are supposed to do - research for the betterment of the wasteland.  :) 

## Commit Update
Disallows the Followers to use the ores from the BoS and others and also replaces the lathe with a medical board to prevent printing of nonmedical stuff 😔 

## Changelog
:cl:
add: Put the RnD stuff in the followers.
add: The ability for the Follower's RnD set to create points.
/:cl:

SquishyOranges committed:
Followers RnD addition

FantasticFwoosh commented:
There is a immediate problem, if this connects to the BOS silo by being on the same z level, another faction is going to tap into their resources already competing with Oasis and the Enclave.

I recommend if you're going to do this, remove the BOS silo first, then put a machine frame & the component parts underneath a circuit board ontop of the frame where it was, either a wired up frame or provided a prepped spool of wire.

ogrigbe commented:
Make your own research branch for the followers or we can't add this. Also what Fantastic said about the fact that they'll be taking each others resources.

FantasticFwoosh commented:
> 
> 
> Make your own research branch for the followers or we can't add this. Also what Fantastic said about the fact that they'll be taking each others resources.

Them having a medical department protolathe would severely inhibit their ability to print things even if they shared tech with other factions, without having like a "master controller" research network with everyone getting a seperate thing.

SquishyOranges committed:
followersrndnerf

disallows followers to use ores from bos and disables the chance of printing nonmedical equip
